serverless/installing_serverless/upgrading-serverless.adoc

```diff
@@ -0,0 +1,32 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="upgrading-serverless"]
+= Upgrading {ServerlessProductName}
+include::modules/common-attributes.adoc[]
+:context: upgrading-serverless
+
+toc::[]
+
+If you have previously installed a Technology Preview version of {ServerlessProductName}, follow the instructions in this guide to upgrade to the latest Generally Available version.
+
+In Technology Preview versions of {ServerlessProductName} (versions 1.6.0 and older), the only available Subscription Update Channel was `techpreview`.
+To upgrade to the latest Generally Available version, you must update the channel to the correct channel for your {product-title} version.
+
+.Prerequisites
+* You have installed a Technology Preview version of {ServerlessOperatorName}, and have selected Automatic updates during the installation process.
+* If you have selected Manual updates for the {ServerlessOperatorName} during the installation process
+// what do we need to do here? Anything additional / different?
```

Comment:
@mgencur is the process any different if they chose Manual updates over Automatic?